news/_posts/core-weekly/2020-04-20-coreweekly-16-2020

```diff
@@ -0,0 +1,184 @@
+
+
+## Code changes in the '1.7.6.x' branch
+
+
+### Core
+* [#18659](https://github.com/PrestaShop/PrestaShop/pull/18659): Changelog for 1.7.6.5, by [@sowbiba](https://github.com/sowbiba)
+* [#18651](https://github.com/PrestaShop/PrestaShop/pull/18651): Update dependencies for security reasons, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#18638](https://github.com/PrestaShop/PrestaShop/pull/18638): Update version to 1.7.6.5, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### Back office
+* [#18637](https://github.com/PrestaShop/PrestaShop/pull/18637): Fix sidebar not displayed in BO Add employee page, by [@Progi1984](https://github.com/Progi1984)
+* [#18607](https://github.com/PrestaShop/PrestaShop/pull/18607): Fix wrong number of "Last emails" in BO - Customer View page. Thank you [@PululuK](https://github.com/PululuK)
+
+
+### Front office
+* [#18633](https://github.com/PrestaShop/PrestaShop/pull/18633): Convert cart rule value when order currency is different, by [@sowbiba](https://github.com/sowbiba)
+
+
+## Code changes in modules, themes & tools
+
+
+### QA nightly results
+* [#35](https://github.com/PrestaShop/QANightlyResults/pull/35): refactoring and adding brwoser and campaign data everywhere, by [@SimonGrn](https://github.com/SimonGrn)
+* [#32](https://github.com/PrestaShop/QANightlyResults/pull/32): Update version, by [@SimonGrn](https://github.com/SimonGrn)
+* [#31](https://github.com/PrestaShop/QANightlyResults/pull/31): Develop, by [@SimonGrn](https://github.com/SimonGrn)
+* [#30](https://github.com/PrestaShop/QANightlyResults/pull/30): better handling of GCPURL, by [@SimonGrn](https://github.com/SimonGrn)
+* [#29](https://github.com/PrestaShop/QANightlyResults/pull/29): add GCPURL in vhost example file, by [@SimonGrn](https://github.com/SimonGrn)
+* [#28](https://github.com/PrestaShop/QANightlyResults/pull/28): add default value for GCPURL, by [@SimonGrn](https://github.com/SimonGrn)
+* [#27](https://github.com/PrestaShop/QANightlyResults/pull/27): add info about composer in readme, by [@SimonGrn](https://github.com/SimonGrn)
+
+
+### nightly-board
+* [#21](https://github.com/PrestaShop/nightly-board/pull/21): Update README.md, by [@SimonGrn](https://github.com/SimonGrn)
+* [#17](https://github.com/PrestaShop/nightly-board/pull/17): Remove unused console log, by [@NeOMakinG](https://github.com/NeOMakinG)
+* [#16](https://github.com/PrestaShop/nightly-board/pull/16): Fix some bugs on suites, by [@NeOMakinG](https://github.com/NeOMakinG)
+* [#15](https://github.com/PrestaShop/nightly-board/pull/15): Set passed toggle to fault by default and adjust loaders, by [@NeOMakinG](https://github.com/NeOMakinG)
+* [#14](https://github.com/PrestaShop/nightly-board/pull/14): Revert test CI, by [@NeOMakinG](https://github.com/NeOMakinG)
+* [#13](https://github.com/PrestaShop/nightly-board/pull/13): Test CI, by [@NeOMakinG](https://github.com/NeOMakinG)
```

Comment:
May be to remove ? Ping @NeOMakinG

Reply:
why should we remove that ?

Reply:
It's test & revert :p

Reply:
Oh god, can't we exclude pull request from nightly ? This will be insane I did so much PR after CI

Or I'll rename these maybe..